README.md

```diff
@@ -79,10 +89,11 @@ Write your code here:
 
 Describe the differences between unit and functional testing. What type of testing is RSpec and why?
 
-Your answer:
+Unit testing test methods in a class and all dependent mock ups. Testing is done on an individual unit.
+Functional units test many methods and interacts with dependent databases or web services. It is also called Integration test (or testing). It tests a part of a functionality in a system
+RSpec  is defect testing framework. It tests domain specific languages which makes it easier to write test.
```

Comment:
:+1:

RSpec can be used to test Domain Specific Languages (DSLs), but I think you meant to say here that _is_ a DSL, specifically written to make it easier to write tests.